src/packer.pkr.hcl

```diff
@@ -75,9 +75,17 @@ data "amazon-ami" "windows_server_2022" {
   region      = var.build_region
 }
 
+
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
 source "amazon-ebs" "windows" {
+  ami_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/xvda"
+    encrypted             = true
+    volume_size           = 500
+    volume_type           = "gp3"
+  }
```

Comment:
You shouldn't need this block if the settings are identical to those in `launch_block_device_mappings`:
```suggestion
```

Reply:
Resolved in commit 8eb3ad1.